CHANGELOG.md

```diff
@@ -125,6 +124,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Code() method from the Request interface (#158)
 - `Schema` field from the `Connection` struct (#7)
 - `OkCode` and `PushCode` constants (#237)
+- Move SSL support to a separate module `go-tlsdialer` (#301)
```

Comment:
It was simply
> ### Removed
> - IPROTO constants

for moving IPROTO constants to a separate package. I think here we too should simply
> ### Removed
> - SSL support (#301)

Reply:
Ok, updated.